test/controllers/declaration/ConsignmentReferencesControllerSpec.scala

```diff
@@ -36,15 +39,13 @@ class ConsignmentReferencesControllerSpec
   import ConsignmentReferencesControllerSpec._
 
   override def beforeEach() {
-    super.beforeEach()
```

Comment:
Por que?

Reply:
_The Assassination of `super.beforeEach()` by the Coward @PauloPortugal_ 

`super.beforeEach()` is promoting inheritance, which with the advent of the functional paradigm has lost some of it's reputation, some languages are today even designed not allow inheritance (such as `Go` or `Clojure`, although they still allow polymorphism)

And that is why Scala loses the appeal. It allows you to write OOP and Functional all in the same place, a bit like Java 8-12.

This is also saying, we should not allow generic behaviour to be available in traits/parent classes.